pilot/pkg/bootstrap/server.go

```diff
@@ -275,9 +275,6 @@ func (s *Server) initClusterRegistries(args *PilotArgs) (err error) {
 			s.clusterStore); err != nil {
 			return err
 		}
-	} else if err = clusterregistry.ReadClustersV2(s.kubeClient,
```

Comment:
@sbezverk @sdake  if this code is removed, then how to handle the case that the secret was created prior to create the istio control panel?

Reply:
Ah, I see, once the pilot is start, it will detect the secret and add the cluster.

Reply:
@gyliu513 exactly, controller will get notification for every already existing secrets matching List/Watch criteria.